src/extension.ts

```diff
@@ -114,7 +115,7 @@ export async function activate(context: ExtensionContext) {
           workspace.getConfiguration("metals").get("javaHome")
         );
         return fetchAndLaunchMetals(context, javaHome);
-      } catch (err) {
+      } catch (err: any) {
```

Comment:
This should not be needed:
```suggestion
      } catch (err) {
```